tools/wake/main.cpp

```diff
@@ -208,13 +210,7 @@ void inspect_database(const CommandLineOptions &clo, Database &db, const std::st
     collect_ands.push_back({"endtime = 0"});
   }
 
-  // Convert the collected parts into a meaningful query
-  if (collect_ands.empty()) {
-    std::cerr << "No filters were applied" << std::endl;
-    exit(1);
-  }
```

Comment:
Do we want to/can we keep this check with a longer boolean?  Does it actually matter since the "Available wake targets" listing will be printed if we don't specify a filter?
```diff
-  if (collect_ands.empty()) {
+  if (collect_ands.empty() && collect_input_ands.empty() && collect_output_ands.empty()) {
```

Reply:
we can go three routes here.

1) add the check back
2) just let the "available wake targets" message show
3) make it so "render flags" are a "no filter" query. aka `wake --timeline` gives all jobs in the database

I originally went with 1 because it would have been extra code to manually remove the `WHERE` from the query but I had to do it anyways for this change so I'm tempted to do 3

Reply:
3 is how I expect that to work since I very rarely use `--timeline`, but I'm not sure it's as intuitive for `--metadata` -- then again, without it we don't have a way of getting *all* the data from the DB at once.  I don't have a strong preference otherwise, so that last realization has me tipping toward 3.

Reply:
I'm in favor of 3.